### PR TITLE
TVR-7261 Document that Concur Request v1, v3, v3.1 don't support company-level token

### PR DESCRIPTION
**Pull request by psoum**
update on preview, Request API v1 and v3 reference documents

## Pull Request Checklist

**Complete and check all items that are applicable.**

- [x] Proofread documentation changes for spelling and grammatical errors
- [ ] Used Markdown code blocks for all applicable examples using code
- [ ] Used relative links when linking to other documentation on Developer Center
- [ ] If creating new API reference pages, followed the guidelines stated [in the wiki](https://github.com/concur/developer.concur.com/wiki/Creating-API-Reference-pages)
- [ ] If moving pages, added [page redirects](https://github.com/concur/developer.concur.com/wiki/Adding-Redirects) so users are redirected to the new page

### CODE REVIEW

**Comment by charlieconcur** [comment]
@psoum This looks fine to me. Please have this peer reviewed by someone on your team, have them give a thumbs up here in the pull request and then I will merge.

File: src/api-reference/request/request.markdown [hunk 1]
```diff
@@ -28,7 +28,7 @@ Name | Type | Format | Description
 -----|------|--------|------------
 `offset`	|	`string`	|	`query`	|	Starting page offset	
 `limit`	|	`Int32`	|	`query`	|	Number of records to return (default 100)	
-`user`	|	`string`	|	`query`	|	The login ID of the user who owns this Request. The user must have the Web Services Admin (Professional) or Can Administer (Standard) user role to use this parameter.	
+`user`	|	`string`	|	`query`	|	The login ID of the user who owns this Request. The user must have the Web Services Admin (Professional) or Can Administer (Standard) user role to use this parameter. Note : company-level token is not supported.
```
Comment (charlieconcur) [inline src/api-reference/request/request.markdown]: Please make sure capitalization matches and remove the space after Note and before the colon.

Comment (psoum) [reply src/api-reference/request/request.markdown]: Thank you Charlie. I make the update asap (will simply remove the "note"), and ask for a peer reviewer on my team for a thumb up.

**Comment by psoum** [comment]
@charlieconcur text has been updated and "thumbed up" by Christelle from my team.
Let me know if you need anything else.
Thanks.

**Pull request merged**